package.json

```
@@ -18,6 +18,7 @@
     "react-icons": "4.2.0",
     "redoc": "2.0.0-rc.53",
     "sass": "1.32.12",
+    "selic": "^0.1.5",
```

O padrão do projeto é deixa pinada a versão
```suggestion
    "selic": "0.1.5",
```